Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Tabular/DriverBlockReason.hs

```diff
@@ -0,0 +1,56 @@
+{-
+ Copyright 2022-23, Juspay India Pvt Ltd
+
+ This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License
+
+ as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program
+
+ is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+
+ or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of
+
+ the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE QuasiQuotes #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE TemplateHaskell #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+module Storage.Tabular.DriverBlockReason where
+
+import qualified Domain.Types.DriverBlockReason as Domain
+import Kernel.Prelude
+import Kernel.Storage.Esqueleto
+
+mkPersist
+  defaultSqlSettings
+  [defaultQQ|
+    DriverBlockReasonT sql=driver_block_reason
+      reasonCode Text
```

Comment:
is this table like a config for blocking ? like we will send these to frontend and based on which one is selected we will block for the specified time ?